octoprint_mrbeam/__init__.py

```diff
@@ -1345,33 +1344,28 @@ def takeUndistortedPictureForInitialCalibration(self):
     @octoprint.plugin.BlueprintPlugin.route(
         "/on_camera_picture_transfer", methods=["GET"]
     )
-    @calibration_tool_mode_only
```

Comment:
now it's open to the world. should be: restricted access OR calibration tool mode